test/unit/@polkadot-api.mock.ts

```diff
@@ -44,4 +45,10 @@ export const signAndSendWithPaymentInfo = {
   paymentInfo(pair: any) { // eslint-disable-line
     return mockFunction();
   }
+}
+
+export const signAndSendReturnToHuman = {
+  signAndSend(pair: any, nonce: any) {
+    return toHumanMock;
+  }
```

Comment:
Why do you need this? Other tests doesn't.

Reply:
Because in some function in service-request.js there aren't callback in signAndSend. other signAndSend mock called callback and it break unit test because callback function is null.

Reply:
Every `command` needs a callback available. Because it can be set to optional.